### ISSUE TITLE
Warning method is immutable

### ISSUE THREAD

**Issue by chigia001** [jfmengels/eslint-plugin-lodash-fp#33]
There is a lot of method in lodash that can mutate the input params like:
_.fill
_.pull
_.assign
...

But these method in lodash/fp will create new object. so these function is very useless without assignment:
For example these code should give errors;

```
var test = {};
_.assign(test, {text: 'test'};
console.log(test.text === 'test');
```

**Comment by jfmengels** [comment]
Hi @chigia001! That sounds a lot like what I attempted to do in https://github.com/jfmengels/eslint-plugin-lodash-fp/blob/master/docs/rules/no-unused-result.md. let me know if turning that one on solves your problems.

**Comment by chigia001** [comment]
Hi @jfmengels ,

That is awesome, I missed that rule.

Thank alot

**Comment by jfmengels** [comment]
;)
